configs/jester.cfg

```diff
@@ -0,0 +1,18 @@
+# input loading config
+# Not relevant for local CSV, but mandatory options.
+load_input_path: /home/ec2-user/cirrus/examples/ml/tests/test_mf/nf_parsed
```

Comment:
If these are not needed can you write NA instead?

Reply:
Changed to NA where I could. For s3_size, I changed it to the smallest non-zero value that is divisible by minibatch_size.